plugin/src/test/java/org/opensearch/ml/rest/RestMLGuardrailsIT.java

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.rest;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.hc.core5.http.HttpEntity;
+import org.apache.hc.core5.http.HttpHeaders;
+import org.apache.hc.core5.http.message.BasicHeader;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.ExpectedException;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+import org.opensearch.core.rest.RestStatus;
+import org.opensearch.ml.common.MLTaskState;
+import org.opensearch.ml.utils.TestData;
+import org.opensearch.ml.utils.TestHelper;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+public class RestMLGuardrailsIT extends MLCommonsRestTestCase {
+
+    final String OPENAI_KEY = System.getenv("OPENAI_KEY");
+    final String COHERE_KEY = System.getenv("COHERE_KEY");
+
+    final String completionModelConnectorEntity = "{\n"
```

Comment:
Move this to Utility class of TestHelper under utils folder?

Reply:
Yeah, will remove in next pr.